test/zzz-correct-index-names.sh

```diff
@@ -24,7 +24,15 @@ es_pod=$( get_es_pod es )
 es_ops_pod=$( get_es_pod es-ops )
 es_ops_pod=${es_ops_pod:-$es_pod}
 
-for project in default openshift openshift-infra ; do
+all=$(oc get project -o jsonpath={.items[*].metadata.name})
+read -r -d '' SCRIPT<< EOF
+projects = [p for p in "$all".split(' ') if p.startswith('openshift-') or p == 'default']
+print " ".join(projects)
+EOF
+
+INFRA_PROJECTS=$(python -x "${SCRIPT}")
```

Comment:
I don't think you want to get the list of projects from the system like this.  I think you want to use `OCP_OPERATIONS_PROJECTS` as the source of project names and patterns to check.

The trick is - how to get this list in this context?  If the env var is set in the fluentd daemonset, you could get it like this:
```
oc set env ds/logging-fluentd --list | grep and sed for OCP_OPERATIONS_PROJECTS
```
if it isn't set in the ds, then you could find the pid of the fluentd process, and do something like this:
```
sudo cat /proc/$pid/environ | tr '\0' '\n' | grep and sed for OCP_OPERATIONS_PROJECTS
```

or ....

Reply:
@richm  I image going forward the pattern will be anything prefixed with 'openshift' should go to the operations index.  Do we not want to insulate ourselves from other changes?  For instance the openshift-template-broker is also a namespace I am seeing.  Like logging, shouldnt that really go to  some infra index?  In the long term we should probably redefine IMO '.operations' to be an alias and infra projects should maybe go into an 'infra' prefixed index.  Short term, maybe your solution is better; i defer to you.

Reply:
> anything prefixed with 'openshift' should go to the operations index.

and `kube` too?  Yes, openshift-template-broker should go into .operations.

My "solution" is that the script that checks to see if the data in es is correct should use the logging configuration of which projects and patterns to use, rather than just use the list of projects returned by `oc get projects`.